cosmo_tester/test_suites/cluster/conftest.py

```diff
@@ -348,18 +357,24 @@ def _bootstrap_manager_node(node, mgr_num, dbs, brokers, skip_bootstrap_list,
     if dbs:
         node.additional_install_config['postgresql_server'] = {
             'ca_path': node.remote_ca,
-            'cluster': {'nodes': []},
+            'cluster': {'nodes': {}},
         }
         node.additional_install_config['postgresql_client'] = {
             'server_username': 'postgres',
             'server_password': dbs[0].pg_password,
         }
 
         if len(dbs) > 1:
-            node.additional_install_config['postgresql_server'][
-                'cluster'][
-                'nodes'] = [str(db.private_ip_address) for db in dbs
-                            if db.friendly_name not in skip_bootstrap_list]
+            db_nodes = {
```

Comment:
Given that you're using this logic in a couple of places and it's now a step more complex than it used to be, would it be worth factoring it out?

Reply:
Discussed in slack: Given the slight differences and this similar code only being in two places it's probably best left for the moment.